include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -100,10 +100,10 @@ def TTG_SelectOp : TTG_Op<"select", [Pure, Elementwise,
   let description = [{}];
 
   let arguments = (ins TT_BoolLike:$condition,
-                       TT_Tensor:$true_value,
-                       TT_Tensor:$false_value);
+                       TT_Type:$true_value,
```

Comment:
We can just use arith.select if non-tensor type

Reply:
Reverted changes to `SelectOp`, noting that `arith::SelectOp` didn't work when I wrote this.